[pulsar admin] getPartitionedTopicMetada method support setting auto create topic (#11986)
apache/pulsar: site2/docs/admin-api-topics.md

```

> <!--REST API-->

Reviewer: Here we have `<!--REST API-->`  and `<!--Java-->` methods, do we have `<!--pulsar-admin-->` method?

Author: No,  pulsar-admin does not have this method

> When topic auto-creation is enabled by broker, you can set param `checkAllowAutoCreation = true` to auto create topic.

Reviewer: ```suggestion
When enabling topic auto-creation, you can set the parameter `checkAllowAutoCreation = true` to automatically create a topic.
```

do you mean this? if so, please help update all occurrences.
